### PR TITLE
Add Branch Rules

### PR DESCRIPTION
**Pull request by dainnilsson**

### CODE REVIEW

File: gitbark/cli/__main__.py [hunk 1]
```diff
@@ -292,6 +276,7 @@ def main():
         _add_subcommands(cli)
         cli(obj={})
     except Exception as e:
+        raise e
```
Comment (elibon99) [inline gitbark/cli/__main__.py:279]: I'm guessing this should be removed?f

File: gitbark/cli/__main__.py [hunk 2]
```diff
@@ -138,10 +135,11 @@ def install(ctx):
     if not repo.lookup_branch(BARK_RULES_BRANCH):
         raise CliFail('Error: The "bark_rules" branch has not been created!')
 
-    root_commit = cmd("git", "rev-list", "--max-parents=0", BARK_RULES_BRANCH)[0]
+    root_commit_hash = cmd("git", "rev-list", "--max-parents=0", BARK_RULES_BRANCH)[0]
+    root_commit = Commit(bytes.fromhex(root_commit_hash), repo)
     if root_commit != project.bootstrap:
         click.echo(
-            f"The bootstrap commit ({root_commit}) of the branch_rules "
+            f"The bootstrap commit ({root_commit.hash.hex()}) of the branch_rules "
```
Comment (elibon99) [inline gitbark/cli/__main__.py:142]: "branch_rules" should be changed to "bark_rules". This is not due to  theses changes (it has been like this for a while), but would be nice to use a constant instead.

File: gitbark/cli/__main__.py (hunk 1 above)
Comment (elibon99) [inline gitbark/cli/__main__.py:279]: I'm guessing this should be removed?

**Review by elibon99 — CHANGES_REQUESTED** [review]

File: gitbark/cli/__main__.py (hunk 1 above)
Comment (dainnilsson) [reply gitbark/cli/__main__.py]: Yeah. We should probably improve the standard error output here to be more descriptive.

**Review by elibon99 — APPROVED** [review]

**Review by elibon99 — APPROVED** [review]

**Pull request merged**